### ISSUE TITLE
Full screen icon not visible on iPhone

### ISSUE THREAD

**Issue by intermedion** [ergec/jQuery-for-PhotoSwipe#1]
The lightbox full screen icon is not visible on iPhone Chrome or Safari browsers when using an actual iPhone device. Rotating the device from portrait to landscape orientation automatically goes in full screen mode. Tapping on the icon location, when in full screen mode, exists full screen and makes browser top nav bar visible. The share and close buttons are visible, as well as the captions. The navigation arrows are also missing, even when there is space to the left and right of the lightbox image and the caption and the other icons are visible (numbering, links, and close icons). 

The above can be verified on the ergec/jQuery-for-PhotoSwipe site demo. They are also apparent on my own site, so it is not only this demo.

The icon is visible for all available devices when the lightbox is opened in the Chrome Developer Tools/Toggle Device Toolbar.

**Comment by ergec** [comment]
My code just triggers the original plugin. Can you confirm if you experience the same problem on this site. This is the original plugin. http://photoswipe.com

**Comment by intermedion** [comment]
Indeed, I did not test it there, but I did now. The issue is indeed with PhotoSwipe. I will post the issue there.

By the way, you did excellent work with your version. The PhotoSwipe developer apparently is not open to the user suggestions to change the PS code to remove the need for img dimensions.

I made a request to add jqPhotoswipe to cdnjs.com but so far they did not, as apparently the usage does not meet their guidelines.
